gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSimpleExtractor.java

```diff
@@ -26,8 +27,11 @@
  */
 public class KafkaSimpleExtractor extends KafkaExtractor<String, byte[]> {
```

Comment:
Can this class and `KafkaSimpleJsonExtractor` be deprecated now that there is `KafkaDeserializerExtractor`?